pulser-core/pulser/channels/base_channel.py

```diff
@@ -18,7 +18,16 @@
 import warnings
 from abc import ABC, abstractmethod
 from dataclasses import MISSING, dataclass, field, fields
-from typing import Any, Literal, Optional, Type, TypeVar, cast, get_args
+from typing import (
+    Any,
+    Collection,
```

Comment:
nit: I think this should be imported from `collections.abc` instead

Reply:
Indeed ! Did not know that :) https://stackoverflow.com/questions/75285711/difference-between-collections-abc-sequence-and-typing-sequence